app/workers/pager_duty/poll_maintenance_windows.rb

```diff
@@ -6,6 +6,7 @@ module PagerDuty
   class PollMaintenanceWindows
     include Sidekiq::Worker
     include SentryLogging
+    sidekiq_options retry: 3
```

Comment:
How about just one retry @annaswims (or none). If this runs every 3m, then we'll have 4 failures every 3m when reachability is not ephemeral. I get 1 retry for ephemeral stuff. 

Is the 3m scheduled frequency good enough?